environment/src/main/resources/schema/app/20220419144557_CB-16699_Bring_your_own_DNS_zone_-_Rename_endpoint_body_field,_step_2.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-16699 Bring your own DNS zone - Rename endpoint body field, step 2
+-- Migration SQL that makes the change goes here.
+ALTER TABLE environment_network DROP COLUMN IF EXISTS privatednszoneid;
```

Comment:
Is it in production use already? 
Because dropping and recreating the column will cause data loss. Have you considered using the update of the column type in place with a single SQL?
`ALTER TABLE ... ALTER COLUMN .... SET DATA TYPE`

Reply:
Thank you for the suggestion.
The column is already in production, but was never used.
This is a 2-step rename. 

Step 1 was:
- introduce a new column, databasePrivateDnsZoneId, and the code uses that only. 
- the code in production has only filled data into column databasePrivateDnsZoneId, as the feature itself and the first part of the rename arrived together on prod with 2.56

Step 2, i.e. the curent PR:
- deletes the column privateDnsZoneId

But I agree, the rename would have been a better, one-step solution.